node-js-basics-101/node_modules/node-static/lib/node-static.js

```diff
@@ -0,0 +1,393 @@
+var fs     = require('fs')
+  , events = require('events')
```

Comment:
__[jshint]__
* Comma warnings can be turned off with 'laxcomma'.
* Misleading line break before ','; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->